### PR TITLE
Fix Keyboard and Input elements

### PR DESCRIPTION
**Pull request by francescochiapello**
## Description

- Receive keyboard formats from element props
- Apply style and textStyle to Input element

### CODE REVIEW

**Review by cee-money — CHANGES_REQUESTED** [review]
Thanks for contributing @francescochiapello 🚀 👏 

I just suggested a couple changes. Would you mind filling out the PR description with a few more details to help reviewers understand your main goal with this PR? I see that we wanted to be able to style the text in the `Input` component, but I'm less clear on what we're tackling with the `Keyboard` changes. Thanks again!

File: packages/solid/components/Input/Input.types.ts [hunk 1]
```diff
@@ -37,6 +37,16 @@ export interface InputProps extends UIComponentProps, InputStyleProperties {
    */
   titleSignal: Signal<string>;
 
+  /**
+   * general styles applied to input `View` container
+   */
+  style?: unknown;
+
```
[the remark this answers is not in the document]
Comment (francescochiapello) [reply packages/solid/components/Input/Input.types.ts:44]: I'm not sure about that, I mean that `style` prop is not recognised by ts.

style

[the remark this answers is not in the document]
Comment (cee-money) [reply packages/solid/components/Input/Input.types.ts:44]: what version of `@lightningtv/solid-ui` do you happen to be on? i'm not seeing this type error when using the latest [`@lightningtv/solid-ui@0.29.1`](https://github.com/rdkcentral/solid-ui/releases/tag/%40lightningtv%2Fsolid-ui%400.29.1)

File: packages/solid/components/Input/Input.types.ts [hunk 2]
```diff
@@ -37,6 +37,16 @@ export interface InputProps extends UIComponentProps, InputStyleProperties {
...
+  /**
+   * specific styles applied to input `Text` item
+   */
+  textStyle?: TextStyles;
```
Comment (anthony9187) [inline packages/solid/components/Input/Input.types.ts:48]: can we update `@lightningtv/solid` import to include `TextStyles` as well? This isn't currently imported

**Review by anthony9187 — CHANGES_REQUESTED** [review]
@francescochiapello thanks again for taking your time to contribute to our project! Just one more request - can you update the PR title to use the following format:
```
fix: <description>
```

our versioning system uses conventional commits to handle version bumps and it'll break without the proper casing/punctuation

**Comment by anthony9187** [comment]
Hi @francescochiapello! This has been sitting stale for a while so I'm gonna go ahead and close it for now. If you're interested in re-opening this please feel free to do so, or reach out to us either via an issue or discord. Thanks!

**Pull request closed without merging**